tmpl/blog/functoria.md

```diff
@@ -0,0 +1,153 @@
+
+For the last two months, I have been at OCaml Labs for "holidays" with the grand task
+of "fixing the mirage tool".
+
+I'm happy to present [Functoria](https://github.com/Drup/Functoria), a library to create arbitrary mirage-like DSLs. Functoria is independent from Mirage and will replace all the core engine that was bolted on the mirage tool until now.
```

Comment:
better to give the `mirage/Functoria` link rather than `drup/functoria`?
also `Mirage-like` rather than `mirage-like`